scripts/extract_modem_trace.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+from elftools.elf.elffile import ELFFile
+import argparse
+import struct
+
+
+# /**
```

Comment:
It would be good with a comment explaining why we have this here (it's useful, just to give the first-time reader context)